backend/app/models/credentials.py

```
@@ -7,8 +7,12 @@
 
 
 class CredsBase(SQLModel):
-    organization_id: int = Field(foreign_key="organization.id")
-    project_id: Optional[int] = Field(default=None, foreign_key="project.id")
+    organization_id: int = Field(
+        foreign_key="organization.id", nullable=False, ondelete="CASCADE"
+    )
+    project_id: Optional[int] = Field(
```

Ruff (0.12.2)

Use `X | None` for type annotations

Convert to `X | None`

(UP045)